apps/docs/workflows.mdx

```diff
@@ -0,0 +1,499 @@
+---
+title: 'Codemod CLI'
```

Comment:
I think the title of the page should be Codemod Workflow. We need a master doc about Codemod CLI and Codemod workflow is one of the sub-pages.

Another feature of the Codemod CLI that we need to cover is [jssg](https://github.com/codemod-com/codemod/tree/main/crates/cli#jssg-javascript-ast-grep)